services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js

```diff
@@ -45,6 +48,19 @@ const applyTransaction = async (blockHeader, tx, events, dbTrx) => {
 		...tx.params,
 	};
 
+	const filterInitializeUserAccountEvent = events
+		.find(event => event.name === EVENT_NAME_INITIALIZE_USER_ACCOUNT
+			&& event.data.address === tx.params.recipientAddress);
+
+	if (filterInitializeUserAccountEvent) {
+		const formattedTransaction = await requestConnector('formatTransaction', {
+			transaction: tx,
+			additionalFee: filterInitializeUserAccountEvent.data.initializationFee,
+		});
+
+		tx.minFee = formattedTransaction ? formattedTransaction.minFee : tx.minFee;
```

Comment:
Same for other places.
```suggestion
		tx.minFee = formattedTransaction.minFee;
```